moulinette/actionsmap.py

```diff
@@ -511,8 +517,13 @@ def generate_cache(klass, namespaces=None):
             with open(am_file, 'r') as f:
                 actionsmaps[n] = yaml.load(f)
 
+            # clean old cached files
+            for i in os.listdir('%s/actionsmap/' % pkg.cachedir):
+                if i.endswith(".pkl"):
```

Comment:
If you want to, that really a matter of taste here.

I find the way it's currently written to be easier to understand since it really says what it does (while glob isn't necessarily obvious for everyone) but I don't have a very strong opinion here (while I do prefer code maintainability and readability)